plugins/Name_UpperCase.py

```diff
@@ -22,6 +22,10 @@
 from plugins.Plugin import Plugin
 import regex as re
 
+# Whitelist of allowed capitals by country code
+UpperCase_WhiteList = {
+    "FR": ["CNFPT", "COSEC", "EHPAD", "MACIF", "MEDEF", "URSSAF"]
+    }
 
```

Comment:
Please, can you align to start of the line.

Reply:
fixed